README.md

```diff
@@ -1,80 +1,90 @@
 [![Build Status](https://travis-ci.org/k8sp/sextant.svg?branch=master)](https://travis-ci.org/k8sp/sextant.svg?branch=master)
 
-# sextant
+#Sextant
 <img src="logo/Sextant.png" width="250">
 
-sextant 提供了可以通过PXE全自动化安装初始化一个CoreOS+kubernetes集群。
+Sextant provides a tool that initialize a cluster with CoreOS and Kubernetes crossing PXE technology.
 
-## 环境准备
-bootstrapper需要运行在一台服务器上(以下称bootstrapper server)，满足以下的几个要求：
+# Enviroment Prepare
+Bootstrapper will running on a machine(bootstrapper server), which need to meet the following requirements
 
-1. 待初始化的kubernetes机器需要和bootstrapper server保持网络连通
-1. bootstrapper server是一台安装有docker daemon(***1.11以上版本***)的Linux服务器
-1. 拥有bootstrapper server的root权限
-1. 配置bootstrapper server的/etc/hosts文件，增加hostname的解析：```127.0.0.1  bootstrapper```
+1. The kubernetes machine to be initialized need to be connected with bootstrapper server.
+2. Bootstrapper server is a linux server with `docker daemon`(more than 1.11 version) installed.
+3. Root permissions with bootstrapper server.
```

Comment:
The step 4 was implemented with https://github.com/k8sp/sextant/blob/develop/start_bootstrapper_container.sh#L38